doc/radosgw/rgw-cache.rst

```diff
@@ -90,7 +90,7 @@ $ sudo ln -sf /usr/local/openresty/bin/openresty /usr/bin/nginx
 
 Put in-place your Nginx configuration files and edit them according to your environment:
 
-All Nginx conf files are under: https://github.com/ceph/ceph/tree/master/examples/rgw-cache
+All Nginx conf files are under: https://github.com/ceph/ceph/tree/master/examples/rgw/rgw-cache
```

Comment:
Since it's being touched anyway (here and in two other files):
```suggestion
All Nginx conf files are under: https://github.com/ceph/ceph/tree/main/examples/rgw/rgw-cache
```